experiments/experiments-impl/src/main/java/com/duckduckgo/experiments/impl/store/ExperimentsDataStore.kt

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (c) 2023 DuckDuckGo
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.duckduckgo.experiments.impl.store
+
+import android.content.Context
+import android.content.SharedPreferences
+import androidx.core.content.edit
+import com.duckduckgo.di.scopes.AppScope
+import com.squareup.anvil.annotations.ContributesBinding
+import javax.inject.Inject
+
+interface ExperimentsDataStore {
+    var variantManagerConfigDownloaded: Boolean
+}
```

Comment:
I don't think we need this store. If we don't have remote variants, we assign default

Reply:
Outdated too. I removed this part when I needed to download remote config before atb is initialised